src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/PerformanceAnalyzerApp.java

```diff
@@ -66,6 +67,7 @@
 
   private static final int EXCEPTION_QUEUE_LENGTH = 1;
   public static final String QUERY_URL = "/_opendistro/_performanceanalyzer/metrics";
+  public static final String BATCH_METRICS_URL = "/_opendistro/_performanceanalyzer/batch";
```

Comment:
Please change the README accordingly to document the new url and details for this

Reply:
Agreed, documentation has been added to the README in the performance-analyzer repository.